Core/Security/VOMS.py

```diff
@@ -81,6 +87,24 @@ def getVOMSProxyFQAN( self, proxy ):
     return self.getVOMSProxyInfo( proxy, "fqan" )
 
   def getVOMSProxyInfo( self, proxy, option = False ):
+    """ Returns information about a proxy certificate (both grid and voms).
+        Available information is:
+          1. Full (grid)voms-proxy-info output
+          2. Proxy Certificate Timeleft in seconds (the output is an int)
+          3. DN
```

Comment:
What should be the convention for the arguments description ? This one or 
:param string param_name: parameter description